src/sentry/notifications/notifications/activity/escalating.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from typing import Any, Mapping
+
+from sentry.services.hybrid_cloud.actor import RpcActor
+from sentry.types.integrations import ExternalProviders
+
+from .base import GroupActivityNotification
+
+
+class EscalatingActivityNotification(GroupActivityNotification):
+    message_builder = "SlackNotificationsMessageBuilder"
```

Comment:
Not your fault but this is kinda confusing it's called Slack message builder but it works for Slack and MSTeams